packages/rancher-monitoring/generated-changes/patch/Chart.yaml.patch

```diff
@@ -16,7 +16,16 @@
  apiVersion: v2
  appVersion: 0.46.0
  dependencies:
-@@ -59,19 +68,17 @@
+@@ -38,7 +47,7 @@
+ - condition: prometheus-adapter.enabled
+   name: prometheus-adapter
+   repository: file://./charts/prometheus-adapter
+-- condition: nodeExporter.enabled
++- condition: nodeExporter.enabled,hardenedNodeExporter.enabled
```

Comment:
Turns out it doesn't quite work like that:
```
Condition - The condition field holds one or more YAML paths (delimited by commas). If this path exists in the top parent's values and resolves to a boolean value, the chart will be enabled or disabled based on that boolean value. Only the first valid path found in the list is evaluated and if no paths exist then the condition has no effect.
```
If the first condition exists, and is falsely then the dependency won't be installed.

Reply:
@dbason thanks for confirming this. I wasn't aware that the first valid path would be taken, even if it was false.

Instead of using tags, would it work if we switch the order of conditions to `hardenedNodeExporter.enabled,nodeExporter.enabled` and removed `hardenedNodeExporter.enabled` from the values.yaml?

Reply:
That's what I was thinking of doing initially, but then hardenedNodeExporter would always be installed unless you explicitly set it to false:
> Only the first valid path found in the list is evaluated and if no paths exist then the condition has no effect.